manifest.toml

```diff
@@ -0,0 +1,83 @@
+packaging_format = 2
+
+id = "garage"
+name = "Garage"
+description.en = "S3 storage"
+description.fr = "stockage S3"
+
+version = "0.8.2~ynh1"
+
+maintainers = ["oiseauroch"]
+
+[upstream]
+license = "AGPL-3.0-only"
+website = "https://garagehq.deuxfleurs.fr/"
+admindoc = "https://garagehq.deuxfleurs.fr/documentation/quick-start/"
+userdoc = "https://garagehq.deuxfleurs.fr/documentation/quick-start/"
+code = "https://git.deuxfleurs.fr/Deuxfleurs/garage"
+
+[integration]
+yunohost = ">= 1.1.18"
+architectures = "all"
+multi_instance = true
+ldap = false
+sso = false
+disk = "50M"
+ram.build = "50M"
+ram.runtime = "50M"
+
+[install]
+    [install.domain]
+    type = "domain"
+    full_domain = true
+
+    [install.rpc_secret]
+    ask.en = "UUID of the network (rpc-secret) "
+    ask.fr = "UUID de l'ilot (rpc-secret)"
+    type = "string"
+    optional = true
+    example = "1799bccfd7411eddcf9ebd316bc1f5287ad12a68094e1c6ac6abde7e6feae1ec"
+
+    [install.bootstrap_peers]
+    ask.en = "friend server id"
+    ask.fr = "serveur ami"
+    type = "string"
+    example = "1799bccfd7411eddcf9ebd316bc1f5287ad12a68094e1c6ac6abde7e6feae1ec@127.0.0.1:3901"
+    optional = true
+
+    [install.weight]
+    ask.en = "number of G to allow"
+    ask.fr = "nombre de G à allouer"
+    type = "string"
```

Comment:
It's a bit funky that this is a string if what's expected is a number, `type` should probably `integer` or `number` (at least one of these exist I think)